db/schema.rb

```diff
@@ -34,6 +34,15 @@
     t.index ["type", "person_id"], name: "index_addresses_on_type_and_person_id", unique: true
   end
 
+  create_table "businesses", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "crime_application_id", null: false
+    t.string "business_type", null: false
```

Comment:
I put `type` as the attribute name in the schema i'm drafting, happy to change the schema to `business_type` or we could change this one to `type` instead

Reply:
I used business_type as its the name of the value object and is the same pattern used for investments, savings etc. Doesn't need to be the same in the schema.